org.speed.dreams.yaml

```diff
@@ -0,0 +1,166 @@
+runtime-version: '1.6'
+sdk: org.freedesktop.Sdk
+command: speed-dreams-2
+separate-locales: false
+finish-args: 
+  - --socket=wayland
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --filesystem=home
+  - --socket=pulseaudio
+  - --share=network
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - '*.a'
+  - '*.la'
+  
+modules:
+  - shared-modules/glu/glu-9.0.0.json
+
+  - name: giflib
+    config-opts:
+      - --prefix=/app
+      - --disable-static
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/sourceforge/giflib/giflib-5.1.4.tar.bz2
+        md5: 2c171ced93c0e83bb09e6ccad8e3ba2b
+  
+  - name: freeglut
+    buildsystem: cmake
+    config-opts:
+    - -DCMAKE_INSTALL_LIBDIR:PATH=/app/lib
+    
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/freeglut/freeglut-3.0.0.tar.gz
+        sha1: fca52242f9344627a30f11487ee42002e6b0dacd
+        
+  - name: jasper
+    buildsystem: cmake
+    config-opts:
+    - -DCMAKE_INSTALL_PREFIX=/app
+    - -DCMAKE_INSTALL_LIBDIR=lib
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DJAS_ENABLE_OPENGL=ON
+    - -DJAS_ENABLE_LIBJPEG=ON
+    - -DJAS_ENABLE_AUTOMATIC_DEPENDENCIES=OFF
+    - -DCMAKE_SKIP_RPATH=ON
+    - -DALLOW_IN_SOURCE_BUILD=ON
+    sources:
+      - type: git
+        url: https://github.com/mdadams/jasper
+        tag: version-2.0.14
+        commit: 1a36ca39da535af2e67848f5f43ffd657746e632
+        
+  - name: openscenegraph
+    buildsystem: cmake
+    config-opts:
+    - -DCMAKE_INSTALL_PREFIX=/app
+    - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-3.4.1.tar.gz
+        sha256: 930eb46f05781a76883ec16c5f49cfb29a059421db131005d75bec4d78401fd5
+  
+  - name: libxmu
+    sources:
+      - type: archive
+        url: https://www.x.org/archive//individual/lib/libXmu-1.1.2.tar.bz2
+        sha256: 756edc7c383254eef8b4e1b733c3bf1dc061b523c9f9833ac7058378b8349d0b
+  
+  - name: plib
+    rm-configure: true
+    config-opts:
+      - CXXFLAGS=-fPIC
+    sources:
+      - type: archive
+        url: http://plib.sourceforge.net/dist/plib-1.8.5.tar.gz
+        sha1: c2cf7e3e1e58f7b63dae4bb21e4fa82c3e4d4cfc
+        
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+
+  - name: enet
+    sources:
+      - type: archive
+        url: http://enet.bespin.org/download/enet-1.3.13.tar.gz
+        sha512: 82501c0248330a3a6bcac907907760e59b18cb662ebb42fe2c91e1557f1bc8ab452bf3fe7477dc01fa7a6e3b6124e1eb8eac46382d5bbebb2394879f9568b5df
+  
+  - name: freealut
+    rm-configure: true
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: http://pkgs.fedoraproject.org/repo/pkgs/freealut/freealut-1.1.0.tar.gz/e089b28a0267faabdb6c079ee173664a/freealut-1.1.0.tar.gz
```

Comment:
Why use the Fedora mirror over here? [Upstream](https://github.com/vancegroup/freealut/releases) seems to have an even newer version.